src/edu/csus/ecs/pc2/core/InternalController.java

```diff
@@ -1058,23 +1060,58 @@ private void insureProfileDirectory(Profile profile) {
             new File(profileDirectory).mkdirs();
         }
     }
+    
+    /**
+     * Load judgements from reject.ini
+     * @param contest
+     * @param cdpConfigDirectory
+     * @return
+     * @throws Exception
+     */
+    protected boolean loadJudgements(IInternalContest contest, File cdpConfigDirectory) throws Exception {
```

Comment:
Maybe I'm missing something about how the PR review process works, but as stated in the previous review comment this looks like it is adding a whole new method which has nothing to do with Issue #119.

Reply:
I did not make this change.